projects/workflows-creator/src/lib/builder/builder.component.ts

```diff
@@ -58,6 +58,9 @@ export class BuilderComponent<E> implements OnInit, OnChanges {
   @Input()
   state: StateMap<RecordOfAnyType> = {};
 
+  @Input()
+  localizedStringMap: {[key: string]: string};
```

Comment:
Instead of passing this everywhere, why not just create a common service, register the map in that service here in this component, than inject that service and use the values from there whenever you need to, or you can even create a simple pipe to fetch that information from that service in html